File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L2/04/singlearray.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const n: number[] = new Array(10);

const readNumber = (): Promise<void> => {
  return new Promise((resolve) => {
    rl.question('Read a number: ', (input) => {
      n[n.length - 1] = parseInt(input);
      n.unshift(n.pop()!);
      resolve();
    });
  });
};

const main = async () => {
  for (let i = 0; i < 10; i++) {
    await readNumber();
  }

  for (let a = 0; a < 10; a++) {
    console.log(n[a]);
  }

  rl.close();
};

main();

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L5/05/singlearray.py
n = [0] * 10

def print_array():
    for a in range(9, -1, -1):
        print(n[a])

def main():
    for i in range(10):
        n[i] = int(input("Read a number: "))
    print_array()

if __name__ == "__main__":
    main()

